src/index.js

```diff
@@ -1,4 +1,6 @@
 import http from 'http';
+// import and instantly use 'process.env.APP_ENV' to define the correct `.env` file to use
```

Comment:
It'd be better if this explanation of what `/server/env` does was at the top of that file instead.